fusor-ember-cli/app/routes/review/installation.js

```diff
@@ -2,8 +2,10 @@ import Ember from 'ember';
 import request from 'ic-ajax';
 
 export default Ember.Route.extend({
-  model() {
-    return this.store.queryRecord('deployment', {id: this.modelFor('deployment').get('id'), sync_openstack: true});
```

Comment:
@cfchase, by deleting the `model` I assume you can just taking the convention of that model will come from it's parent route, right? In this case, it's the model for deployment route, right?